robot-installer.sh

```diff
@@ -19,7 +44,6 @@ function latest_cirrus(){
     echo "$VERSION"
 }
```

Comment:
Does this still make sense to pull from github now that we are installing from pip?